beacon-chain/node/node.go

```diff
@@ -209,10 +212,23 @@ func New(cliCtx *cli.Context, opts ...Option) (*BeaconNode, error) {
 		return nil, err
 	}
 
-	bfs := backfill.NewStatus(beacon.db)
-	if err := bfs.Reload(ctx); err != nil {
+	log.Debugln("Registering P2P Service")
+	if err := beacon.registerP2P(cliCtx); err != nil {
```

Comment:
I double checked and don't think starting p2p before stategen should cause issues, but still something to be wary of if we do have future bug reports

Reply:
good to keep in mind indeed